api.py

```diff
@@ -116,12 +119,20 @@ def exec(data):
 
 def status():
     sts = get_status()
+    # getting reset log file
+    try:
+        with open('/home/pi/log/reset_trigger_service.log', 'r') as log_file:
```

Comment:
We already have a `logs` folder in the backend. Better keep all the logs in the same place.